src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -50,6 +63,14 @@ protected void doFilterInternal(final HttpServletRequest request,
                     timing,
                     contentEncoding,
                     acceptEncoding);
+
+            nakadiKpiPublisher.publish(accessLogEventType, () -> new JSONObject()
+                    .put("method", method)
+                    .put("path", path)
+                    .put("query", query)
+                    .put("app", user)
```

Comment:
We cannot use it for statistics in datalake.  
We need to add some hashing. As here https://techjira.zalando.net/browse/ARUHA-1321